internal/app/exporter/ec2.go

```diff
@@ -1,13 +1,11 @@
 package exporter
 
 import (
-	"regexp"
-
 	"github.com/qonto/prometheus-rds-exporter/internal/app/rds"
 	"golang.org/x/exp/slices"
 )
 
-func getUniqTypeAndIdentifiers(instances map[string]rds.RdsInstanceMetrics) ([]string, []string) {
+func GetUniqTypeAndIdentifiers(instances map[string]rds.RdsInstanceMetrics) ([]string, []string) {
```

Comment:
```suggestion
func getUniqTypeAndIdentifiers(instances map[string]rds.RdsInstanceMetrics) ([]string, []string) {
```
Seems like it does not need to be exported.

Reply:
As discussed, I moved them to private function.

Helper:

> Go allows to place tests in a separate package with suffix _test. For example, tests for store package can be in the same package or in the package store_test. In the second case, you have to import the source code into tests so only exported things are available.
> The linter reports if a test is in a package without suffix _test. If you really need to test unexported function, then put the test into file XXX_internal_test.go. The linter skips such files by default. It also skips the file export_test.go by default (see the last article below).
> https://github.com/maratori/testpackage